TIMELINE.md

```diff
@@ -0,0 +1,13 @@
+# Timeline
+
+We plan two stages, each with different focus areas:
+* Stage 1, February 2024: StatelessNet will focus on demonstrating functional readiness of upcoming changes. MVP version of stateless validation with minimal optimization will be released first and followed by iterations on feature completeness.
+* Stage 2, March 2024: Once all critical features for stateless validation are stabilized and corner cases are addressed, StatelessNet will move on to the next stage, focusing on performance and reliability optimization. Multiple rounds of load tests and stress tests will be conducted to test the limit of stateless validation and identify bottlenecks. Engineering focus will be around optimizations on performance and reliability.
+
+## Notable dates
+Here are couple notable (approximate) dates during duration of the program
+* February 1st: Stake Wars IV Stage 1 starts
+* February 14th: Bug bounty reward program starts
+* Feb 29th - March 3rd: NEAR Protocol shares Stateless Validation and Stake Wars with the blockchain community at EthDenver.
+* March 1st: Sake Wars IV Stage 2 starts. Reward programs start for traffic generation and running nodes.
```

Comment:
typo Sake Wars
I would totally participate in that too though ;)

Reply:
We should organise them as well for sure